doc/api/diagnostics_channel.md

```diff
@@ -1117,6 +1117,64 @@ Emitted when server receives a request.
 
 Emitted when server sends a response.
 
+#### Modules
+
+`module.require.start`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
```

Comment:
I don't think `name` is actually part of the `event` (same below).
If it is part of the event tests should be adapted to verify this (currently `name` is added in the event handler there).